mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -171,6 +172,9 @@ public void testJoinTables() throws IOException {
   public void testJoinTablesSupportedTypes() throws IOException {
     for (int i = 0; i < SUPPORTED_TYPES.size(); i++) {
       Type type = SUPPORTED_TYPES.get(i);
+      if (type == Types.UUIDType.get() && fileFormat == FileFormat.PARQUET) {
```

Comment:
Please add TODO with the pointer for the issue raised with Parquet and UUID (#1881)

Reply:
Done.